src/mem/simple_mem.hh

```diff
@@ -188,7 +200,13 @@ class SimpleMemory : public AbstractMemory
     Tick recvAtomic(PacketPtr pkt);
     Tick recvAtomicBackdoor(PacketPtr pkt, MemBackdoorPtr &_backdoor);
     void recvFunctional(PacketPtr pkt);
+<<<<<<< HEAD
```

Comment:
This isn't right ;).

Reply:
I don't remember that I added this to the commit. Will fix this.